### PR TITLE
hotfix for truncating gd api values

### PR DESCRIPTION
**Pull request by marcievanauken**
Please review code changes for gd api truncate hotfix

### CODE REVIEW

File: gdBackend.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
 	insertGifts(gifts);
 }
 
+function truncateGifts(gifts, dbLengths){
+	console.log(dbLengths);
+
+	for (gift in gifts){
+		gift.lengthTooLong = true;
+		if (gifts[gift].lengthTooLong){
+			truncateVals(gift);
```
Comment (afarhood) [inline gdBackend.js:14]: please change to truncateValues()

File: gdBackend.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
 function gdProcess(){
 	getGifts(endpoint, timeParams);
 	validateGifts(gifts);
+	truncateGifts(gifts, dbLengths);
```
Comment (afarhood) [inline gdBackend.js:4]: this looks good.

**Review by afarhood — COMMENTED** [review]
I reviewed each sections and added my comments. Please optimize the function and change some labels (these comments are all for demo purposes)

File: gdBackend.js (hunk 1 above)
Comment (marcievanauken) [reply gdBackend.js:14]: sure thing

File: gdBackend.js (hunk 2 above)
Comment (afarhood) [reply gdBackend.js:4]: testing unresolve

**Review by afarhood — APPROVED** [review]
everything looks good, thanks!

**Pull request merged**